src/envoy/http/authn/sample/APToken/guide.txt

```diff
@@ -0,0 +1,18 @@
+This is a guide of sending an example exchanged token to
```

Comment:
To echo the previous point about adding test, why developer needs to follow this series of steps to maintain/debug the code if they can do so with a simple "go test" command? Not to mention, the instruction is not clear what significant one needs to look for to see if APToken was handled correctly.

Reply:
The "go test" for running the Istio authn integration tests are in a different repository istio/istio under the path https://github.com/istio/istio/mixer/test/client. To add an integration test for authenticating an exchange token, this PR should be merged first, then the SHA in the istio.deps of the istio/istio proxy should be updated to the new SHA of istio-proxy, and finally an integration test for an exchanged token can be added. I have created an issue https://github.com/istio/istio/issues/10514 to add an integration test in the istio/istio repository and assigned it to myself.

Reply:
I mean https://github.com/istio/proxy/tree/master/test/integration. The one in istio repo is a plus, but not really necessary.

Reply:
Done: integration tests have been added to https://github.com/istio/proxy/tree/master/test/integration.